modules/react/common/lib/styles/cxFn.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable guard-for-in */
+import {ColorStyleProps} from '@workday/canvas-kit-labs-react/common';
+
+export function cxFn<S extends CXStyleProps>(styles: S) {
+  let cxStyles = {};
+
+  for (const key in styles) {
+    if (key in colorProps) {
+      const value = styles[key as keyof ColorStyleProps];
+      const styleFn = colorProps[key as keyof ColorStyleProps];
+      const cxStyle = styleFn(value);
+      cxStyles = {...cxStyles, ...cxStyle};
+    }
+
+    if (key in depthProps) {
+      const value = styles[key as keyof DepthStyleProps];
+      const styleFn = depthProps[key as keyof DepthStyleProps];
+      const cxStyle = styleFn(value);
+      cxStyles = {...cxStyles, ...cxStyle};
+    }
+  }
+
+  return cxStyles;
+}
```

Comment:
The return type of this function is an empty object. Should it be something else? Like `CSSObject` or `CSSType`?

Reply:
gonna delete this for Alans PR